src/main/java/com/sericulture/authentication/model/entity/RoleMaster.java

```diff
@@ -0,0 +1,40 @@
+package com.sericulture.authentication.model.entity;
+
+import jakarta.persistence.*;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.AvoidStarImportCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Using the '.*' form of import should be avoided - jakarta.persistence.*.